prow/plugins/trigger/ic_test.go

```diff
@@ -174,14 +282,14 @@ func TestHandleIssueComment(t *testing.T) {
 					Name:      "job",
 					AlwaysRun: true,
 					Context:   "pull-job",
-					Trigger:   "@k8s-bot test this",
+					Trigger:   "(^@k8s-bot test this\r?$|^/test\r?$)",
```

Comment:
Actually I'm pretty sure the submit queue puts a bunch of stuff at the end, so we should explicitly validate that the current submit queue phrase continues to trigger.